src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -2781,21 +2978,47 @@ HRESULT CProfilerManager::InitializeForAttach(
 
     m_bAttach = true;
 
+    if (pvClientData == nullptr ||
+        cbClientData == 0)
+    {
+        // No configuration provided
+        return S_FALSE;
+    }
+
     vector<CComPtr<CConfigurationSource>> configSources;
 
-    // TODO: Parse configuration information from pvClientData.
+    // WCHAR = 2 bytes, hence we halve the byte count.
+    UINT charCount = cbClientData / 2;
```

Comment:
>2 [](start = 36, length = 1)

sizeof(WCHAR) or (sizeof(WCHAR) / sizeof(byte)) if you really want to get technical.

Reply:
Probably could just make this a constant, something like: const size_t WCharSizeInBytes = sizeof(WCHAR) / sizeof(byte);

---
In reply to: [386708337](https://github.com/microsoft/CLRInstrumentationEngine/pull/216#discussion_r386708337) [](ancestors = 386708337)